openedx/core/djangoapps/ccxcon/api.py

```diff
@@ -0,0 +1,146 @@
+"""
+Module containing API functions for the CCXCon
+"""
+
+import logging
+import urlparse
+from oauthlib.oauth2 import BackendApplicationClient
+from requests_oauthlib import OAuth2Session
+from rest_framework.status import HTTP_201_CREATED
+
+from django.core.validators import URLValidator
+from django.core.exceptions import ValidationError
+from django.http import Http404
+
+from courseware.courses import get_course_by_id
+from instructor.access import list_with_level
+from openedx.core.djangoapps.models.course_details import CourseDetails
+from student.models import anonymous_id_for_user
+from .models import CCXCon
+
+log = logging.getLogger(__name__)
+
+
+CCXCON_COURSEXS_URL = '/api/v1/coursexs/'
```

Comment:
edx has a very strict policy about settings and I don't think this is going to become one (unless they asme me to move it there)